impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -210,6 +211,23 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
             clientConfig.setProxyPassword(props.get(DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME));
         }
 
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME))) {
+            clientConfig.setAuthorizationMode(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME))) {
+            clientConfig.setClientId(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME));
```

Comment:
CLIENT_CLIENT looks funny, probably should rename this to `DEFAULT_CLIENT_ID_PROPERTY_NAME`
even if the property is `okta.client.client-id`